pulseapi/utility/userpermissions.py

```diff
@@ -11,17 +11,18 @@ def is_staff_address(email):
     if email is None:
         return False
 
-    parts = email.split('@')
-    domain = parts[1]
+    if '@' in email:
```

Comment:
The old code crashed Django when I tried to test with `admin/admin` rather than `admin@example.com/admin`, so this fix is part of this PR in the sense that without it, you can't log into the admin for testing using a simple local account